src/Microsoft.PowerShell.GlobalTool.Shim/GlobalToolShim.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+
+namespace Microsoft.PowerShell.GlobalTool.Shim
+{
+    /// <summary>
+    /// Shim layer to chose the appropriate runtime for PowerShell Core DotNet Global tool.
+    /// </summary>
+    public class EntryPoint
+    {
+        private const string PwshDllName = "pwsh.dll";
+
+        private const string WinFolderName = "win";
+
+        private const string UnixFolderName = "unix";
+
+        /// <summary>
+        /// Entry point for the global tool.
+        /// </summary>
+        /// <param name="args">Arguments passed to the global tool.</param>
+        public static void Main(string[] args)
+        {
+            var currentPath = new FileInfo(System.Reflection.Assembly.GetEntryAssembly().Location).Directory.FullName;
+            var isWindows = Environment.OSVersion.Platform == System.PlatformID.Win32NT;
```

Comment:
In CorePsPlatform.cs we have
```c#
        public static bool IsWindows
        {
            get
            {
                return RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
            }
        }
```

Reply:
Did not want to create a dependency from this `dll` to `SMA.dll` for just one function.

Reply:
Sorry, I meant that we could use `RuntimeInformation.IsOSPlatform(OSPlatform.Windows)`.

Reply:
Changed to use `RuntimeInformation`